### ISSUE TITLE
ipywidgets Dropdown appears as drop-up when published to dashboards

### ISSUE THREAD

**Issue by rs2** [jupyter-incubator/dashboards#278]
The `Dropdown` ipywidget is actually displayed as "drop-up" when published as a dashboard...

I'm guessing an `ipywidgets` component needs to be patched. Which one?

**Comment by parente** [comment]
I think ipywidgets is still using bootstrap components and bootstrap makes some dynamic decisions about whether to pop-up or down based on screen real estate. Can you include a screenshot with the popup open in dashboard view?

**Comment by jhpedemonte** [comment]
Noticed this too. For some reason, when clicking on the dropdown, the class `dropup` gets added, making the menu appear above the widget.

**Comment by jhpedemonte** [comment]
Should be fixed the PR above (in the `dashboards_server` project).

**Comment by rs2** [comment]
@parente @jhpedemonte What actually makes `dropdowns` behave as such and not `dropups` is having the following lines removed from `node_modules/jupyter-dashboards-server/public/css/style.css`:
```css
/* Around line 3619 */
.dropup .caret,
/* ... */
.dropup .dropdown-menu,
```

What change is required to do so?

**Comment by jhpedemonte** [comment]
This was an issue in Dashboards Server that was fixed by jupyter-incubator/dashboards_server#295. But I don't think we've cut a new release with this fix yet. If you are able to, run Dashboards Server from the latest code.
